### PR TITLE
Use ESModule output in production, fix devserver

### PR DESCRIPTION
**Pull request by AAGaming00**
explod

### CODE REVIEW

**Comment by Vendicated** [comment]
wrong

**Comment by AAGaming00** [comment]
insan

File: webpack.config.d/esmodule.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+if (config.mode == "production") {
+    (config.experiments = config.experiments || {}).outputModule = true;
```
Comment (Vendicated) [inline webpack.config.d/esmodule.js:2]: you will 
```suggestion
    (config.experiments ??= {}).outputModule = true;
```

Comment (AAGaming00) [reply webpack.config.d/esmodule.js]: > you will

guhhhh i did

**Comment by zt64** [comment]
you love

**Comment by AAGaming00** [comment]
yop

**Pull request merged**